numpyro/infer/autoguide.py

```diff
@@ -1,12 +1,7 @@
 # Copyright Contributors to the Pyro project.
 # SPDX-License-Identifier: Apache-2.0
 
```

Comment:
I would suggest that we do this in a way where we raise a warning first so that existing code doesn't immediately break, like we did in Pyro:

```python

from numpyro.infer.autoguide import *  # noqa F403

warnings.warn("numpyro.contrib.autoguide has moved to numpyro.infer.autoguide. "
              "The contrib alias will stop working in future versions.",
              FutureWarning)
```

Reply:
Oh, good idea!!!! I have to make another PR blocked because of this issue. With this, I won't have to do that. Thanks!